### ISSUE TITLE
A simple tutorial for a newbie

### ISSUE THREAD

**Issue by Shavinder** [bennylope/django-organizations#193]
Hello there,
I think this is not the correct way of contacting you, but I could not find any other channel.

I am looking for simple tutorial on how to actually use django-organizations in my project. I could not find any tutorials and documentation is not very helpful.

**Comment by rkpip3** [comment]
plz provide simple tutorial..

**Comment by florianmartens** [comment]
True. This project is quite hard to get started with. The docs are very, very dense. 

But still thanks for creating and releasing this under MIT.

**Comment by obvionaoe** [comment]
Any news on this?

**Comment by sebanism** [comment]
Commenting as a bump. Would be great to see some additional examples with django v4.2. Struggling with URLs not working with abstract models. Documentation is ok, but hard to tell if the use case I'm trying to apply this package isn't what it was built for. Used the example from it source created 4 years ago, and the URLs there are also not working with the abstract models.. Could just be me, but not many references available to compare against. Would love to see this succeed.
